src/plugins/mpath.c

```diff
@@ -345,7 +346,7 @@ static gchar** get_map_deps (const gchar *map_name, guint64 *n_deps, GError **er
 /**
  * bd_mpath_is_mpath_member:
  * @device: device to test
- * @error: (out): place to store error (if any)
+ * @error: (out) (allow-none): place to store error (if any)
  *
  * Returns: %TRUE if the device is a multipath member, %FALSE if not or an error
```

Comment:
This API will then hide the difference between an error scenario and valid `FALSE` result.

Reply:
I asked about this in the [original issue](https://github.com/storaged-project/libblockdev/issues/527) :-)

We have two options:

1. Add an out argument.
2. Tell user to provide error if they are interested in the difference between "not member" and "error occurred".

From the API user view I don't think there is a big difference between these two situations, you need to provide an extra out variable either way. If you think 2 is the better way, we can do that, but probably in a separate PR later (to avoid changing API for UDisks for as long as possible).

Reply:
That was merely two years ago! :-) My point is, when we have a chance right now to break the API, whether to change this or not...

E.g. the prototype would then look like this:
```C
gboolean bd_mpath_is_mpath_member (const gchar *device, gboolean *result, GError **error)
```

Reply:
Ok, lets deal with these functions separately so we can merge this without changing API. The functions that need to be changed are tracked in the original issue.

Reply:
OK, let's merge this one now.